libs/shared/db/mysql/account/src/lib/types.ts

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Cart } from './cart';
+
+export type CartFields = Pick<
```

Comment:
I couldn't find a better way of doing this. Open to ideas if anyone has any?

Reply:
Looks correct to me. What were you hoping to improve/what was your issue with this approach?